loadtest/control/gencontroller/controller.go

```diff
@@ -192,6 +192,13 @@ func (c *GenController) Run() {
 			frequency:  int(c.config.NumFollowedThreads),
 			idleTimeMs: 1000,
 		},
+		// This action does not generate data, but is needed for all other actions
+		// to have data to work with
+		"getPosts": {
+			run:        c.getPosts,
+			frequency:  100,
```

Comment:
100 is fine for lower number of posts. But for a batch size of 20M, this should be 10k. We can do a test run and then tune if needed.

Reply:
Yeah, can we perhaps make this somewhat proportional with a low enough constant? Like 1/1000 of total posts or something. We can play with it and see I suppose but I'd avoid the static value.

Reply:
Yeah, something like max(100, totalPosts/1000) should be good.

Reply:
Good. Done in https://github.com/mattermost/mattermost-load-test-ng/pull/609/commits/749a330ca14fdb93891ad7c33d4bc70a7f84c626